build-patched-kao-kcmo-images.sh

```diff
@@ -95,7 +95,10 @@ function patch_and_push_image() {
     git checkout --track origin/rhaos-${OCP_VERSION}-rhel-8
     git merge --no-ff -m "Merge commit ${vcs_ref} into rhaos-${OCP_VERSION}-rhel-8" -m "MaxFileSize: 104857600" ${vcs_ref}
     git push origin HEAD
-    rhpkg container-build --target crc-1-rhel-8-candidate
+    # If brew build already exist for the release don't rebuild it again
```

Comment:
When does this happen? I thought we'd only run this when there are new cluster-kube-apiserver-operator / cluster-kube-controller-manager-operator builds?

Reply:
it happen when a new version of openshift release (mostly z stream) where these component might not even change so this way when we build the image it should not  fail.

Reply:
> Ah they don't rebuild everything even when it's unchanged? ok!

Yes in the past that what I observed.